tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -218,7 +234,33 @@ class RecommendationSystemTest : public BackendTest {
     EE_.clear();
     mod_.clear();
     result = nullptr;
-    bindings_.clear();
+    bindings_->clear();
+
+    auto *traceContext = context_.getTraceContext();
+
+    if (traceContext) {
+      const ::testing::TestInfo *const test_info =
+          ::testing::UnitTest::GetInstance()->current_test_info();
+      std::string testName(test_info->name());
+      std::string testCaseName(test_info->test_case_name());
+
+      for (auto &c : testName) {
+        if (c == '/') {
+          c = '-';
+        }
+      }
+
+      for (auto &c : testCaseName) {
+        if (c == '/') {
+          c = '-';
+        }
+      }
+
+      auto traceFileName =
+          strFormat("%s-%s-%s.json", traceFile.getValue().c_str(),
```

Comment:
Add a leading `/` here so that it still works as expected whether or not the path passed in on the command line ends in `/`.